web/oss/src/styles/auth.css

```
@@ -0,0 +1,327 @@
+/* Sign-in / sign-up redesign — scoped brand tokens and component styles.
+   These values come straight from the design handoff (design_handoff_signin_redesign)
+   and are intentionally raw brand tokens rather than the app's antd palette: the auth
+   screen is a bare marketing surface with its own literary type + yellow keycap language.
+   Everything is scoped under .auth-redesign so nothing leaks into the product app. */
+
+/* ── Fraunces (headlines only) ───────────────────────────────────────────── */
+@font-face {
+    font-family: "Fraunces";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Fraunces" (font-family-name-quotes)

(font-family-name-quotes)